lang/python/sqlparse/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
```

Comment:
generally speaking, i would split this `sqlparse` package into a separate PR;
it looks pretty ready

Reply:
Done [that](https://github.com/openwrt/packages/pull/10375).